app/views/users/index.html.erb

```diff
@@ -1 +1,12 @@
-<h1>Here's a list of Users</h1>
+<%= stylesheet_link_tag 'users_index' %>
+<section class='users-section'>
+<% @users.each do |user| %>
+    <div class='user-container'>
+        <%= image_tag(user.photo) %>
+        <div class='user-info'>
+            <h2 class='username'><%= user.name %></h2>
+            <p class='number-of-posts'>Number of posts: <%= user.posts_counter %></p>
+        </div>
```

Comment:
- [ ] As required for the project, kindly ensure that when clicking on each **user** takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)